android/sdl_android/src/main/java/com/smartdevicelink/transport/SdlRouterService.java

```diff
@@ -1121,7 +1124,7 @@ public void onCreate() {
 
 
         if (!initCheck()) { // Run checks on process and permissions
-            deployNextRouterService();
+            deployNextRouterService(receivedVehicleType);
```

Comment:
@kostyaBoss On line 1127 you don't need to pass in `receivedVehicleType` to the method `deployNextRouterService`.  It is a top level variable.